src/chronometer.js

```diff
@@ -1,33 +1,54 @@
 class Chronometer {
   constructor() {
     // ... your code goes here
+    this.currentTime = 0;
+    this.intervalId = null;
+    
   }
 
   start(printTimeCallback) {
     // ... your code goes here
+    if (printTimeCallback) {
+      this.intervalId = setInterval(printTimeCallback, 1000);
+     }
+     else {
+      this.intervalId = setInterval(() => {
+        this.currentTime++;
+      }, 1000)
+    }
   }
```

Comment:
```suggestion
    this.intervalId = setInterval(() => {
    this.currentTime++; 
   if (printTimeCallback) {
    printTimeCallback()
   } 
  }, 10);
```
Esto antes que usarlo con un setInterval, es mejor que ejecuteis la function, tal cual habeis preguntando pero era solo ejecutarla :D